<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="2">
     <widget class="QLabel" name="process_label">
      <property name="text">
       <string>Process</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="minimise_checkbox">
      <property name="text">
       <string>Minimise</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="variables_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>12</height>
       </size>
      </property>
      <property name="text">
       <string>Select Variables</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="material_label">
      <property name="text">
       <string>Material</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="model_label">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="maximise_checkbox">
      <property name="text">
       <string>Maximise</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="runsno_label">
      <property name="text">
       <string>Number of Runs</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="searchmethod_label">
      <property name="text">
       <string>Search Method</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="goal_label">
      <property name="text">
       <string>Goal</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="2">
     <widget class="QPushButton" name="next_button">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QCheckBox" name="setto_checkbox">
      <property name="text">
       <string>Set to</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QDoubleSpinBox" name="setto_value"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="3">
     <widget class="QComboBox" name="process_dropdown"/>
    </item>
    <item row="1" column="3" colspan="3">
     <widget class="QComboBox" name="material_dropdown"/>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QComboBox" name="model_dropdown"/>
    </item>
    <item row="3" column="3" colspan="3">
     <widget class="QComboBox" name="goal_dropdown"/>
    </item>
    <item row="5" column="3" colspan="3">
     <widget class="QComboBox" name="searchmethod_dropdown"/>
    </item>
    <item row="6" column="3" colspan="3">
     <widget class="QSpinBox" name="runsno_value">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="6">
     <widget class="QListWidget" name="variables_listwidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16777214</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16777214</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
